src/connection.c

```diff
@@ -207,12 +205,12 @@ int createNopollConnection(noPollCtx *ctx)
 					ParodusError("Received Unauthorized response with status: %d\n", status);
 					//Get new token and update auth header
 					char new_token[4096] ={'\0'};
-					if (strlen(get_parodus_cfg()->token_acquisition_script) >0)
+					if (strlen(get_parodus_cfg()->token_acquisition_script) >0) {
 						createNewAuthToken(new_token,sizeof(new_token));
```

Comment:
I think there is a latent bug here because we don't update the upstream token in the configuration.  So the next time we need to re-connect we will use the old token.